### DISCUSSION TITLE
Sticky notifications do not allow to skip task today

### DISCUSSION THREAD

**Discussion by sm-g** [iSoron/uhabits#307] [Feature Requests]
You only can mark as completed or delay it.

Workaround: mark as completed, open app and uncheck

**Comment by iSoron** [comment]
In my mind, the point of sticky notifications is exactly that you cannot dismiss them, unless you perform the habit. You can still snooze them, but they will come back after a while. If you want to be able to dismiss notifications without performing the habit, you can disable the sticky notifications option and simply swipe the notification away.

**Comment by MazeWebArchitect** [comment]
First, I'd like to thank you for your engagement. 
Second, I agree with the issue commitee:
When the app's push notification is set to permanent (which is very useful, so I don't want to set it to swipeable), it makes (imho) sense to be given the option to mark it as not-completed as a third option. Especially when one wants to establish new habits, the option to "fail" is indispensable.

**Comment by psampathkumar** [comment]
It would be nice if we can leave sticky notifications for certain tasks but not all.
